build.gradle

```diff
@@ -4,26 +4,26 @@ buildscript {
     ext.compile_sdk_version = 33
     ext.target_sdk_version = 28
     ext.min_sdk_version = 21
-    ext.kotlin_version = '1.7.10'
-    ext.room_version = '2.4.3'
+    ext.kotlin_version = '1.8.0'
+    ext.room_version = '2.5.0'
     ext.core_ktx_version = '1.8.0'
-    ext.material_version = '1.6.1'
+    ext.material_version = '1.8.0'
     ext.retrofit_version = '2.9.0'
     ext.okhttp_version = '4.10.0'
     ext.mockito_kotlin_version = '2.2.0'
     ext.junit_version = '4.13.2'
-    ext.junit_ext_version = '1.1.3'
+    ext.junit_ext_version = '1.1.5'
     ext.rx_android_version = '2.1.1'
     ext.lifecycle_version = '2.5.1'
     ext.livedata_version = '2.5.1'
-    ext.coroutines_version = '1.6.2'
+    ext.coroutines_version = '1.6.4'
     ext.gson_version = '2.9.1'
-    ext.androidx_appcompat_version = '1.5.0'
+    ext.androidx_appcompat_version = '1.6.1'
     ext.androidx_test_runner = '1.4.0'
-    ext.androidx_test_espresso = '3.4.0'
+    ext.androidx_test_espresso = '3.5.1'
```

Comment:
Bude ta změna espressa OK? Na hlavní appce je freeznutá nějaká 3.3.0 nebo tak něco.

Reply:
Okej, nechá starou pro jistotu

Reply:
Jo, zkoušel jsem zvednout a po chybách jsem to vzdal